### PR TITLE
fix(templates): make base partial responsive

### PR DESCRIPTION
**Pull request by ckarlof**
These are changes suggested by Jessilyn to the base template to make it responsive. It's been roughly tested in Email on Acid, but probably needs more.

### CODE REVIEW

File: partials/base/base.html [hunk 1]
```diff
@@ -3,10 +3,17 @@
...
+  <style type="text/css">
+   @media only screen and (max-width: 480px) {
+     div[class="mobiledevicesupport"] table.container {
+       width: 320px !important;
+     }
+   }
+	 </style>
 </head>
-
+<div class="mobiledevicesupport">
```
Comment (vladikoff) [inline partials/base/base.html]: @ckarlof a div outside the body does not seem right, should we put that into the body?

**Comment by ckarlof** [comment]
There some centering problems with the changes.

File: partials/base/base.html (hunk 1 above)
Comment (ckarlof) [inline partials/base/base.html]: lol, indeed. These changes were suggested by jdavis from the email team late on Friday, and I was mainly just trying to capture and distribute them to the wider team.

**Comment by vladikoff** [comment]
@ryanfeeley our emails already fit well in mobile: 



Let me know if setting a 480px media query is necessary...

**Pull request closed without merging**